cmake/FreecivDependencies.cmake

```diff
@@ -21,7 +21,7 @@ check_function_exists(at_quick_exit HAVE_AT_QUICK_EXIT)
 find_package(Python3 REQUIRED)
 
 # Required as the main networking and utility library
-find_package(Qt5 5.15 COMPONENTS Core Gui Network REQUIRED)
+find_package(Qt6 6.7 COMPONENTS Core Gui Network REQUIRED)
```

Comment:
Qt 6.8 is LTS

Reply:
I'll likely need to drop to 6.6 for Ubuntu 22.10

Reply:
There was a suggestion to go all the way back to 6.5 since it's an LTS release. However, no distro seems to ship it anymore.